src/Stan/Observation.hs

```diff
@@ -129,10 +130,13 @@ prettyShowObservation Observation{..} = unlines $
             arrow = srcSpanEndCol observationLoc - start - 1
 
     solution :: [Text]
-    solution = case inspectionSolution inspection of
-        []   -> []
-        sols -> formatWith [italic, green] "Possible solution:" :
+    solution
+        | isHidden toggleSolution = []
+        | null sols = []
```

Comment:
Do you think it makes sense to simplify condition or it's better to specify it on separate lines for better visibility?
```suggestion
        | isHidden toggleSolution || null sols = []
```